src/scripts/components/LoginPage.tsx

```diff
@@ -56,6 +58,13 @@ export default function LoginPage ({ source, isLoggedIn, isPopup }: LoginProps)
     if (source === 'quick-start' || loggedIn) document.title = 'Welcome to Toggl Button';
   }, [source, loading, error, loggedIn]);
 
+  const onLoginError = React.useCallback(
+    (error: string) => {
+      setState({ loading: false, loggedIn: false, error });
```

Comment:
Should bugsnag be notified or does that get handled by existing stuff?

in any case, I guess we want to know about both original failures and then if the secondary form continues to fail 😬

Reply:
This existing effect should report the error

https://github.com/toggl/toggl-button/blob/2b8b1da4d5ad986b422dc01f88b3446c199f0e3f/src/scripts/components/LoginPage.tsx#L58-L62